src/aof.c

```diff
@@ -792,6 +792,38 @@ int openNewIncrAofForAppend(void) {
     return C_OK;
 }
 
+/* Called in `rewriteAppendOnlyFileBackground` to open a new temp INCR AOF when in 
+ * AOF_WAIT_REWRITE state, the AOF information will not be added to server.aof_manifest 
+ * and persisted to manifest file.
+ * */
+int openNewTempIncrAofForAppend() {
```

Comment:
Maybe worth unifying this with `openNewIncrAofForAppend()` with some boolean `temp` flag, just to avoid code duplication.

Reply:
i agree.. even if it doesn't really remove a lot of code duplication, it's easier to read and modify when you clearly see what's running in one mode and what's in the other (by looking at the `if`s)

Reply:
Agree.